Automattic/newspack-popups
# Add Exporter class

### All Submissions:






### Changes proposed in this Pull Request:



### How to test the changes in this Pull Request:

1. Make sure to run `composer install` to update the autoloader
2. Tests are passing?
3. Test it on a site with existing Campaigns, prompts and segments
4. Run `wp newspack-popups export`
5. Check the output the the json file created
6. It should contain all your data
7. Check if the `totals` have the correct information with the total number of exported items
8. Modify your database to create an invalid data

* For example, set the `overlay_size` to something that is not a valid value.
* Set the 'options->excluded_tags` with an ID of a tag that does not exist.

9. Export again
10. Check that the invalid items are not exported
11. Check the errors array containing the respective errors


### Other information:
